<launch>
    <node pkg="sort" name="SortTracking" type="sort" output="screen">
        <param name="view_image"                value="true"/>
        <!-- track param -->
        <param name="min_hits"                  value="3"/>
        <param name="max_age"                   value="120"/>

        <param name="input_topic1"              value="/yolo/bbx"/>
        <param name="input_topic2"              value="/yolo/img"/>
        <param name="output_topic"              value="/result"/>
    </node>

    <!-- <node pkg="rviz" name="rviz" type="rviz" args="-d $(find projection_search)/proj.rviz" required="false"/> -->
</launch>
